scripts/pipeline.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/library.sh"
+
+if [[ -z "${SOURCES_DIR}" ]]; then
+    echo "Missing SOURCES_DIR variable"
+    exit 1
+fi
+
+init
+
+cd "${SOURCES_DIR}"
```

Comment:
Is this on purpose? Maybe pushd ${SOURCES_DIR} and popd would be better approach.

Reply:
Done